listen.gemspec

```diff
@@ -33,6 +33,9 @@ Gem::Specification.new do |s|
   s.add_dependency 'rb-fsevent', '>= 0.9.3'
   s.add_dependency 'rb-inotify', '>= 0.9.7'
 
+  # Used to show warnings at runtime
+  s.add_dependency 'ruby_dep', '~> 1.1'
```

Comment:
My answer wasn't completely correct:

Local installation of the gem takes 2 steps:

1. Building the gem (I can't really fix this, except with the LoadError exception handling to avoid setting the ruby dependency requirement)
2. Installing the gem locally.

The second is where the line helps, because the rake task uses `gem install` and `gem` will try to install `ruby_dep` for you.

But ... since you're building locally, `ruby_dep` should already be installed by Bundler in the previous step.

So it depends whether you're using Bundler or not.